components/BenefitTasks.js

```diff
@@ -32,6 +32,9 @@ export default function BenefitTasks(props) {
                 target={
                   newTabTaskExceptions.includes(task.link) ? '_blank' : '_self'
                 }
+                rel={
+                  props.newTabTaskExceptions ? 'noopener noreferrer' : undefined
```

Comment:
I don't think `props.` should be there right? Just `newTabTaskExceptions`.

Reply:
good catch